src/ore/src/overflowing.rs

```diff
@@ -84,6 +84,12 @@ impl<T: std::fmt::Display> std::fmt::Display for Overflowing<T> {
     }
 }
 
+impl<T: PartialEq<T>> PartialEq<T> for Overflowing<T> {
+    fn eq(&self, other: &T) -> bool {
```

Comment:
Do you mind adding a `#[inline(always)]` here? It's not consistent but my intention was to have it on all functions.